tools/build_config/clone_herbert_branch.ps1

```diff
@@ -0,0 +1,26 @@
+# Clone and build the given Herbert branch. If no branch is given, default to
+# master
+# Pass a specific branch using the -branch parameter, any other arguments are
+# passed verbatim to the Herbert build script
+
+param(
+  [string][Alias("B")]$branch = "master",
+  [string]$build_args = ""
+)
+
+. $PSScriptRoot/powershell_helpers.ps1 <# Imports:
+  Write-And-Invoke
+  Invoke-In-Dir
+#>
```

Comment:
that's confusing comment indenting; maybe would make more sense to thave the Imports Write+Invoke and Invoke-In-Dir comment abouve the ./powershell_helpers.ps1 line?